pkg/domain/entities/images.go

```diff
@@ -171,3 +171,16 @@ type ImagePruneReport struct {
 	Report Report
 	Size   int64
 }
+
+type ImageImportOptions struct {
```

Comment:
Nit: if we comment all this stuff now, we don't need to do it in the future. Let's name it while it's hot?